### ISSUE TITLE
[MultiComboBox]: `readonly` state not read out

### ISSUE THREAD

**Issue by ilhan007** [SAP/ui5-webcomponents#7775]
### Bug Description

_This is issue is created on behalf of SAP Build Processs Automation:_

`readonly` multi combobox is not announced as readonly by the screen reader with VoiceOver on Chrome).
The output on my side does not include the `readonly` state
<img width="599" alt="Screenshot 2023-11-01 at 8 28 45" src="https://github.com/SAP/ui5-webcomponents/assets/15702139/500f21c3-dfc6-4aca-829d-f3f21affd413">

<img width="595" alt="Screenshot 2023-11-01 at 8 28 56" src="https://github.com/SAP/ui5-webcomponents/assets/15702139/86f302c4-d36d-41f8-9e3c-34d6a1c0f66c">

### Affected Component

ui5-multi-combobox

### Steps to Reproduce

1. Open https://ui5-webcomponents-ngx.netlify.app/?path=/docs/ui5-web-components-main-multicombobox--docs
2. Scroll to the instances of the multi combobox after the overview section
3. Turn on VoiceOver
4. Focus the readonly instance

### Priority

None

### UI5 Web Components Version

>=1.16.0

### Browser

Chrome

### Operating System

MacOS

### Additional Context

The issue is experienced with the ui5-webcomponents-ngx wrapper, but it seems it's related to the native web component as the same output is read out in our playground https://sap.github.io/ui5-webcomponents/playground/?path=/story/main-multi-combo-box--basic&args=readonly:!true

### Organization

SAP

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-rl could you take a look at the issue?

**Comment by ndeshev** [comment]
Hello @ilhan007 

We are currently looking into the reported issue.

The issue is reproducible with both VoiceOver on MacOS and NVDA on Windows and we observed the same when testing native HTML elements, for example the codepen snippet linked here - https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-both/

JAWS (I tested 2023 & 2021) announce the `readonly ` state correctly. It could be SR issues, but in any case I will discuss the matter with the accessibility experts and update the issue again.

**Comment by ndeshev** [comment]
Hello,

After discussion with our accessibility experts I will close the ticket because it seems like the issue is in result of deviations in the interpretation of HTML, ARIA, and WCAG standards depending on the combination of browser and screen reader. 

The current recommended testing environment is:
Windows 11
Chrome latest
JAWS 2023

We are aiming to supports other environments to the extend of providing a valid HTML and ARIA implementation following the WCAG standards.

I updated our internal acc wiki pages for the `ui5-combobox` and ui5-`multi-combobox` with the reported issue in order to be monitored in the future.

Kind Regards,
Nikolay
